cloud.foundry.cli/src/test/java/cloud/foundry/cli/operations/ApplicationOperationsTest.java

```diff
@@ -20,13 +28,181 @@
 
 import java.nio.file.Paths;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 
 public class ApplicationOperationsTest {
```

Comment:
A link to the tested class (ApplicationOperations) as it is done in test classes like AllInformationOperationsTest would be nice.

Reply:
done